lib/puppet/parser/functions/generate.rb

```diff
@@ -11,7 +11,7 @@
     generators, so all shell metacharacters are passed directly to
     the generator.") do |args|
 
-      raise Puppet::ParseError, "Generators must be fully qualified" unless Puppet::Util.absolute_path?(args[0])
+      raise Puppet::ParseError, _("Generators must be fully qualified") unless Puppet::Util.absolute_path?(args[0])
```

Comment:
Could you add a note that `fully qualified` here refers to a fully qualified file system path?